app/models/document.rb

```diff
@@ -1,7 +1,12 @@
 class Document < ApplicationRecord
   belongs_to :application
 
-  has_attached_file :attachment, PAPERCLIP_STORAGE_OPTIONS 
+  has_attached_file :attachment
+  # , PAPERCLIP_STORAGE_OPTIONS 
```

Comment:
Keep what was originally there - add a `PAPERCLIP_STORAGE_OPTIONS` constant in the `config/environments/test.rb` file